vsts/dps-e2e.yaml

```diff
@@ -0,0 +1,70 @@
+resources:
+- repo: self
+#Multi-configuration and multi-agent job options are not exported to YAML. Configure these options using documentation guidance: https://docs.microsoft.com/vsts/pipelines/process/phases
+jobs:
+
+- job: 'Test'
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  strategy:
+      matrix:
+#        Python27:
+#          python.version: '2.7'
+#        Python34:
+#          python.version: '3.4'
+#        Python35:
+#          python.version: '3.5'
+#        Python36:
+#          python.version: '3.6'
+        Python37:
+          python.version: '3.7'
+      maxParallel: 1
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python $(python.version)'
+    inputs:
+      versionSpec: '$(python.version)'
+      architecture: 'x64'
+
+  - script: 'python -m pip install --upgrade pip'
+    displayName: 'Update pip'
+
+  - script: 'python -m pip install azure-iot-device'
```

Comment:
It looks like this job will test against the azure-iot-device package from PIP.  Is that what you want?

Reply:
yeah it should not be

---
In reply to: [314835490](https://github.com/Azure/azure-iot-sdk-python-preview/pull/167#discussion_r314835490) [](ancestors = 314835490)